framework/sysdeps/linux/meson.build

```diff
@@ -1,17 +1,26 @@
 # Copyright 2022 Intel Corporation.
 # SPDX-License-Identifier: Apache-2.0
 
+if device_type == 'cpu'
+    if (host_machine.cpu_family() == 'x86_64')
+        framework_files += files(
+            'kvm.c',
+            'interrupt_monitor.cpp',
+            'msr.c',
+        )
+    else
+        framework_files += files(
+            '../generic/kvm.c',
```

Comment:
What requires even the generic version? Ideally we'd not compile anything with "kvm" in the name in the first place.

Reply:
Oh, it's already there.